### ISSUE TITLE
Check the Welcome Notes for 1.3.3. WCAG compliance

### ISSUE THREAD

**Issue by pedr** [laurent22/joplin#11647]
![image](https://github.com/user-attachments/assets/29343e9d-262d-4f39-bf93-416f0ba8c844)
I think we could include in the instruction on how to move the notebook by using only the keyboard in the case above (is it possible?)

**Comment by personalizedrefrigerator** [comment]
> using only the keyboard in the case above (is it possible?)

It should be possible to move notebooks from the context menu.

At least on Windows and Linux, it should be possible to open this menu from the keyboard. On keyboards without a context menu key, I think "shift"-"F10" should open this menu.

**Comment by pedr** [comment]
Makes sense, I forgot about this option, thanks!
